### ISSUE TITLE
Log successful token creations

### ISSUE THREAD

**Issue by hjacobs** [zalando/planb-provider#43]
To help debugging and see how services migrate to JWT tokens.

**Comment by hjacobs** [comment]
I decided on the following log message:
```
INFO: Issued JWT for 'test-service' requested by client /services/test-service_0815
INFO: Issued JWT for '12***89 (15e2b0d3)' requested by client /customers/testclient
```

**Comment by hjacobs** [comment]
We should reconfigure the log to include client IP addresses (don't forget to honor proxies like LBs)

**Comment by hjacobs** [comment]
The client's IP address was implemented with #64.
